pkg/apis/serving/v1alpha1/revision_defaults.go

```diff
@@ -27,7 +27,5 @@ func (r *Revision) SetDefaults() {
 }
 
 func (rs *RevisionSpec) SetDefaults() {
-	if rs.ConcurrencyModel == "" {
-		rs.ConcurrencyModel = RevisionRequestConcurrencyModelMulti
-	}
+	// Nothing to do
```

Comment:
Do you want to copy over `concurrencyModel` to `containerConcurrency` if `concurrencyModel` is set?